### ISSUE TITLE
Filename encryption panic 

### ISSUE THREAD

**Issue by liberianbarcode** [ncw/rclone#784]
Whenever I enable filename encryption for my acd remote I get panic as soon as I try to copy files from my local directory to the remote acd dir.

> What is your rclone version (eg output from `rclone -V`)
    rclone v1.33-63-gace1e21β

> Which OS you are using and how many bits (eg Windows 7, 64 bit)
  Debian Jessie 8.4 64bit

> Which cloud storage system are you using? (eg Google Drive)
Amazon Cloud Drive

> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
rclone copy /localdir/subdir/subsir/ acd:backup/docs

> A log from the command with the `-v` flag (eg output from `rclone -v copy /tmp remote:tmp`)

2016/10/12 02:17:54 rclone: Version "v1.33-63-gace1e21β" starting with parameters ["rclone" "-v" "copy" "/home/user/subdir/subdir/" acd:docs/docs"]

2016/10/12 02:17:55 EME operates on 1 to 128 block-cipher blocks, you passed 137
panic: EME operates on 1 to 128 block-cipher blocks, you passed 137

goroutine 1 [running]:
panic(0x9e95a0, 0xc4201b8950)
        /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/runtime/panic.go:500 +0x1a1
log.Panicf(0xb124f3, 0x3a, 0xc42120e8e8, 0x2, 0x2)
        /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/log/log.go:327 +0xe3
github.com/rfjakob/eme.Transform(0xdf3d20, 0xc420063710, 0xc420080c10, 0x10, 0x10, 0xc4200ffb00, 0x890, 0x900, 0x5301, 0xc4200fb600, ...)
        /home/travis/gopath/src/github.com/rfjakob/eme/eme.go:119 +0x1cc
github.com/ncw/rclone/crypt.(*cipher).encryptSegment(0xc420080bd0, 0xc42123c260, 0x880, 0xc4200ff200, 0x880)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/cipher.go:242 +0xef
github.com/ncw/rclone/crypt.(*cipher).encryptFileName(0xc420080bd0, 0xc421238005, 0x535c, 0xc4201454d9, 0x25)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/cipher.go:278 +0xb4
github.com/ncw/rclone/crypt.(*cipher).EncryptFileName(0xc420080bd0, 0xc421238005, 0x535c, 0xaf0f19, 0x6)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/cipher.go:288 +0xa0
github.com/ncw/rclone/crypt.NewFs(0xc421238000, 0x4, 0xc421238005, 0x535c, 0x4, 0xc421238005, 0x535c, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:76 +0x423
github.com/ncw/rclone/fs.NewFs(0xc421238000, 0x5361, 0x2, 0xc421238000, 0x5361, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc420170a00, 0x4, 0xc420170a05, 0x3270, 0x4, 0xc420170a05, 0x3270, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc420170a00, 0x3275, 0x2, 0xc420170a00, 0x3275, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc420148000, 0x4, 0xc420148005, 0x1dea, 0x4, 0xc420148005, 0x1dea, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc420148000, 0x1def, 0x2, 0xc420148000, 0x1def, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc42015a000, 0x4, 0xc42015a005, 0x114b, 0x4, 0xc42015a005, 0x114b, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc42015a000, 0x1150, 0x2, 0xc42015a000, 0x1150, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc421220a00, 0x4, 0xc421220a05, 0x9aa, 0x4, 0xc421220a05, 0x9aa, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc421220a00, 0x9af, 0x2, 0xc421220a00, 0x9af, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc420012680, 0x4, 0xc420012685, 0x521, 0x4, 0xc420012685, 0x521, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc420012680, 0x526, 0x2, 0xc420012680, 0x526, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc42008ab00, 0x4, 0xc42008ab05, 0x27f, 0x4, 0xc42008ab05, 0x27f, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc42008ab00, 0x284, 0x2, 0xc42008ab00, 0x284, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc4200827e0, 0x4, 0xc4200827e5, 0x110, 0x4, 0xc4200827e5, 0x110, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc4200827e0, 0x115, 0x2, 0xc4200827e0, 0x115, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0xc421210240, 0x4, 0xc421210245, 0x55, 0x4, 0xc421210245, 0x55, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0xc421210240, 0x5a, 0x2, 0xc421210240, 0x5a, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/crypt.NewFs(0x7ffc53a437ea, 0x4, 0x7ffc53a437ef, 0xe, 0x4, 0x7ffc53a437ef, 0xe, 0x0)
        /home/travis/gopath/src/github.com/ncw/rclone/crypt/crypt.go:77 +0x4d2
github.com/ncw/rclone/fs.NewFs(0x7ffc53a437ea, 0x13, 0x10, 0x1, 0xc4200001a0, 0xc420173b68)
        /home/travis/gopath/src/github.com/ncw/rclone/fs/fs.go:447 +0x7d
github.com/ncw/rclone/cmd.newFsDst(0x7ffc53a437ea, 0x13, 0xdf8280, 0xc420138b40)
        /home/travis/gopath/src/github.com/ncw/rclone/cmd/cmd.go:126 +0x4d
github.com/ncw/rclone/cmd.NewFsSrcDst(0xc42017e8a0, 0x2, 0x2, 0xc42017e8a0, 0x2, 0x2, 0xc42017e9a0)
        /home/travis/gopath/src/github.com/ncw/rclone/cmd/cmd.go:136 +0x87
github.com/ncw/rclone/cmd/copy.glob..func1(0xe1eac0, 0xc42017e8a0, 0x2, 0x2)
        /home/travis/gopath/src/github.com/ncw/rclone/cmd/copy/copy.go:58 +0x8d
github.com/spf13/cobra.(*Command).execute(0xe1eac0, 0xc42017e740, 0x2, 0x2, 0xe1eac0, 0xc42017e740)
        /home/travis/gopath/src/github.com/spf13/cobra/command.go:603 +0x439
github.com/spf13/cobra.(*Command).ExecuteC(0xe1de00, 0x20, 0xc42017e680, 0xc420138af0)
        /home/travis/gopath/src/github.com/spf13/cobra/command.go:689 +0x367
github.com/spf13/cobra.(*Command).Execute(0xe1de00, 0x0, 0x40)
        /home/travis/gopath/src/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
        /home/travis/gopath/src/github.com/ncw/rclone/rclone.go:15 +0x31

**Comment by ncw** [comment]
If I'm reading that correctly you have a file for directory name which is 137 AES blocks long, 2176 characters according to the traceback.

That seems unlikely since I think most linux file systems are limited to 255 character file names.

I made an rclone binary which contains an extra log message which looks like this

    2016/10/12 11:03:22 encryptSegment "Makefile" length 8

Can you give it a go and tell me what it says immediately before the crash?

http://pub.rclone.org/rclone-v1.33-63-gace1e21-crypt-log.gz

Thanks

**Comment by liberianbarcode** [comment]
Sorry for the late reply, just came back from work. 

Below is the output from running the DEV binary.
It's not only the copy command but f.e. the mkdir command too,

2016/10/12 17:24:40 rclone: Version "v1.33-DEV" starting with parameters ["rclone" "-v" "mkdir" "acd:directory/subdirectory"]
2016/10/12 17:24:40 encryptSegment "media" length 5
2016/10/12 17:24:40 encryptSegment "movies" length 6
2016/10/12 17:24:41 encryptSegment "3il5vhs9vsrc9g1dvhtrd4mplk" length 26
2016/10/12 17:24:41 encryptSegment "n4o7m5fjh1e31fvs5e8idavo3s" length 26
2016/10/12 17:24:41 encryptSegment "o82eqkgg0d8ann3rfoa1i56iqb31ttctjvckghgmjs0gjaa3ll40" length 52
2016/10/12 17:24:41 encryptSegment "fegd3shph1hrrtq3e81fnl1mq1ev461knqfdcg5jnelrpeoka22g" length 52
2016/10/12 17:24:41 encryptSegment "pdaki9ld8jfckdg5alo8jkd1vc8p2l48ctvlfe574jo1b095faqv5kt2ob1kpbqbbckqccur2v443eucr                                                                                                                        i1l6g72v3urg0tpmarjdk8" length 103
2016/10/12 17:24:41 encryptSegment "33c7ic0mmpeg349j49o406mjdc31fro2faampk178k3g6hvnjkco05ieuu0sg3ggpq3bj8chgpdvgh8a2                                                                                                                        h79m3uh8em27jalnntsano" length 103
2016/10/12 17:24:41 encryptSegment "d9a328m0h04q26in8380qr3cdc0naeikorcevbjqj7vj0374pg56i9vj3p0tsltev5jegq6q2n468b18q                                                                                                                        4fvfu0bg6r02q7tv7pv6imd320fhc9kg6lkdjco6l3bj1vf4ulamnomptoqo7ts03bg0l4g1iotdoj0ubokiqgfoed9q1s2opag" length 180
2016/10/12 17:24:41 encryptSegment "fbh9lj5tda371au4fumjnlbojbclq3t6m0v84vuo37rf6jp7flldn0cn94tc1sg2vpt0nd668i3rkle6m                                                                                                                        a878qt0e8b6hi7pogb8lhv2kpbmmeg2jcgqkf20lbscgf287aqcmahun0ljanlvuv57b8ddmvfo44c72m3d6uarlss72tgr0mig" length 180
2016/10/12 17:24:41 encryptSegment "k9fjmco1r463ud7v9eoo1a4lo5jpgu32vvnnadm5f96pr5gqau0mki6lc62k9ejk3nrbfdjsg86ngtbv7                                                                                                                        nps5qhkq7sgt63hk96o9ph48n4mv5ak04mhuvclrmjno90lcobvr1l6ho6std47hkbkjcrme47min1sp23gp4iu6bqffemdaqecj13udp5i7f03onohmu                                                                                                                        7rk9vlb3kljig1isa9mn81meba6bqu1tr94pb1d4t6tffe20j2vht6gcpvuvp56a58sff48q7t54gorf2o03l6k3krplih96r7epomf0ek8t6g" lengt                                                                                                                        h 308
2016/10/12 17:24:41 encryptSegment "tvppenl7pdce99ib1v6s3llev1pqirus8p9e5kmqper6jq7j9r2cle27tvh89lfqjn04iak0pptugfspj                                                                                                                        1o6n2c9kms681kks79iri08pu5eff92uhso14p52gl00ke45sffskstt771s1vha4npuckltci4m12ultd7vt3od25irprsm15ii55em0i1972qc3fd04                                                                                                                        tcehmnui1q340eq84g0ou57ucuc5i2838ivu1dparijl6a331fkem6eb2m06ierg93lq4psreivatkt1o5ss82st4km4lcriavg052l9d564og" lengt                                                                                                                        h 308
2016/10/12 17:24:41 encryptSegment "iilikmh30a49gv451lfe13bhh5u4j5hvgd9io1tto5u6r1esc7vage8s9eggsliq1mjias3o3depjsfp6                                                                                                                        37npmhkvh49dvqjnvec7im1ffhcr3kf9cbeuebcq4rm3q228e979q87rhj31cro93q4p46mvq9809nt47urhins35ij4f5i5muinehs34smgsvlanjouk                                                                                                                        ig8glteiego2vpqal1c73jb7cf0l8gqfie2pqhkdata2htvkgc2uar7ej7q7ap0neg16g5nebsasm3ehic9rgtta9stq21ujb8qjrb9c3fhq9b92vveo8                                                                                                                        e8mivuobd3pfvadujo4bg400g6l5qsu4quneooajhi8vr68l2ahpsb2hmoplrsu6ihcrkti3maf2v9e379jpi6hbjd9lccumjoqdh4hj16fovlga1pdfm                                                                                                                        4jd92u6drr5oq6sdki3g9lc7lhg87rp744ueoeikkjmaat3plfddufep6f1fjpr2bd9uga2rqi72fn1u" length 512
2016/10/12 17:24:41 encryptSegment "1kh42516cdtmcc1u2phks5mc2vldjmeld4klu3fh5fmq3hgoaqu04achg415ok5p6sml44s90bnq0vu8b                                                                                                                        t48d9kl1dps5lalfqqi21alcqf37gj8o9lts33t0fgkgccvu734m5nhgqs7jb4m3pirbinsilel000an07f904784etihp7eedkv2nefq75ssldcqeno8                                                                                                                        a6hfkrc6itv7dnc94hhb8i0fiujih5kd164p1ah85ka3poirk5o6ggfilekt0pq6rspk7a3nqinpkm0ggcl0u5arqr52v3m3k9adeeekqoqnleg4u0tjn                                                                                                                        d5536jjjmgtpvootvp56i72432fbk60bjhb8ntt7cqv70qj20fv6ugh2cul0krmtvlmjueo2svj1g0if287bbiv3ur5rqbc0jb8fre3igirsguss601p9                                                                                                                        732dj37l8lqk9fng3mddr547l4m73v8s8jquv2d5kralv5b4thg7rirhvrqlkijdicrugb6lvhd9q9gg" length 512
2016/10/12 17:24:41 encryptSegment "81anuheb0mvuj8g6vhm399sfifg8fpjjqpnlqrs0oc1n98hksenff93c27u4uimqb62cm24n242ii689a                                                                                                                        mjag52pod1omf4lqkj0931a52g4e3sb5a0cmgrgphf12c6e4ta2o4fmtrt9todecrjhkrvagr422vdtgokuo5qb69lfknaqe4g937vl352s0q8ju5934l                                                                                                                        ulsn7r45cl9g19lqjgfncp2al6un6bf4sgfd006e4hbbkju1ji49v0sntgs7lqcb5dkr3t79bp7a1g4sbq665ombcrh36nnojv78uerhbuhgl8jgmj3rn                                                                                                                        o5pul5j30qiekvoq4ajc492239rj34bvnou7thi805salevs6lnk1v4748nb1qg0mr467089ka9tthvut7atslam9lr205bbk025gscnr63cduvbq3mqs                                                                                                                        f3km58c6jp2hkdcq2qbbpadsfkaenm297ojnb9uljus51r47oe38gleq0rvjk0o05pj9bkui1jrruht7brslkfpmpu7j75dmk71062fl9apc3o8fvjscf                                                                                                                        ta8cj9bnis63t8519cguqb4fm4pnaoo6nr4bd5v2cl6flm010fr0kg6rvqs9m93vf9s1s6427hr54qnspvhpte900c7536jslj061hnd7829iqnota06i                                                                                                                        ej9i1pe3n9b8bqn0crilabrturbqb68m3mvs98lbpn9ijfn0aaf017ld3bnvje7s36nm0jj8oqv1h9j7j2tlcpgvkkls2impvss3suj5secse7gcd4v7s                                                                                                                        gchsvdqbu856etuocbb88iviltg5j9o7m0bt34nuj406ud7hft3833hldoc4ts" length 845
2016/10/12 17:24:41 encryptSegment "094jvtotnfjqj9889l1bov3te2fj6vbae1e8kl8s291gbddu8jf7ntike4ii97nekfds5pfrlnjolfnq5                                                                                                                        r764elk1cpvgiafdvglff8sam76vo2k8mm1m9crr5va3jkjsccvsv6mqnmk64tsei0g1k0ikric2rl1f75fdoq5bvbqk6q5ii71t2fpn4hma8hgod9110                                                                                                                        thie7j1h6kc692peba766973ro8tjsalb2s1qc2nk0d39stardti7t99v662b0lviukdft3vo8br5t4qibrfj9v7pestun578p3eat4iintaqcon0v9b9                                                                                                                        fgrdi4ug5v259hhh6te4naa542n51o2bqqveig5nao5gp3ius3o47fngeib8h1d7o8fg5clnkimkoc3k0iut3biatvmrmvkca8of7tl2tq1a67o4udksk                                                                                                                        bf0baltctes5vt8hmgu3vj8sst7l9ggk824upvq0u2185afc8lo3f05daqmtegrvtahf6fur6mqh9tu4qa9njmrr3iial6503cjakbd6t1q0u61tlh716                                                                                                                        gka3pailv5972rsdqqaqihe1pmkh7flhg6oqai1ogrdasrg5crqi63fc7r6ka7ocf7uu7tb7vniap35nghit01k2qkdia1jl2kb8iec4v21r1jl8sfce9                                                                                                                        9joocn5gt7feti4h60jeve7hca4gn3m0ubv4tku5o6eo4kgoa7dr77fdigt7flovuj0eetbdvh5k3hvqfmq0bm0v95c754hnd5csvj21lv6d043db3ov3                                                                                                                        k09p465rr89bdgpbrch1ao93vd27votagr6l09srd5dcbrehbts4guoimsfp2k" length 845
2016/10/12 17:24:41 encryptSegment "g7d4rdhvu6rkl0544031eeko7otiji54ln897lm1dtgm2pgvojo46fpisegd7n7iief2dvn7io9jdc0gd                                                                                                                        gmmg2tno0m9lr4s963lfrbjgbkt385343ee7dqv6ullq8rbk2ect0i15psqjgibqllsfs5equ6aqpku0tsb70m5o5b8c0agf0suuc9nfeffjc6kbbdh6c                                                                                                                        u61poapmcor0p8ik9305dvoi4it5tojcp7u4k7275ki5tiaduu1e75rhlkkdjmqk87q64a06m9322clqaiopkfjp87tk7dn6oj60ivm23vm8us4m2g7r7                                                                                                                        hmh6qg0ju9allsqm5g9i2rpuu14osofb6rg1reabnqkk3a5nlg8ng8gagl1bpb6ai2kp5ogpnnhdgq2c2qvjcmt3dddfdbi9c21pdo5dr7bgavofiij4d                                                                                                                        r3f7346q36c0foej8fjtkfd0inots3c5o33jdl7vkut9aaajc694qhvdum511pc3pig4vm0ec56jap9sdp21gld4i3gjl6rhrkbu7ql6rmgit9jtgk84d                                                                                                                        drprpms49p0dknskm4coqsf8nddgcvc2pe4o1ma8e076glcl9didlf8gqoc6uf0lbubo9e66knk14hs1shgp1rb5cl6r9m46ptvvg7qroietorb3u5dnp                                                                                                                        abg9f27vigmu9tlakhjdeg1pk74g3rqfvli57papiti6rs8m3dgvk1ffipei4gc1i68p35vv83ksrlb6hoioj9s5j72v2oce8l1at0mm3el4line7arnk                                                                                                                        d3qek325fb77uf2h27ldc39vs2eu5r59hsnf644h060e7n99f2lk5e7rfklhucibgcdls26mjhsfjmftvq5dutbmnlv6htpfn368d5b5it41ijsuros38                                                                                                                        gvsbffuq4c7a9k19lhg1b8ohro0b20pajnokt263511203lbnd47u2h0qmjpjmhvbfup9e35aganuma6f5oui6i5sttmngj7dtc1ee37as59b0tlfv6ih                                                                                                                        kt36vkfdg3as7p4dq5ctokn2f2regm122fb495fc0vcdollevej0vlft29v9ageqkngv4j9eu2qna2l6qq72falkrtc2iikg0tdsv6a39ick1k8logdpf                                                                                                                        hvove2pb5aknf6bujp31ssu33p6ub09i72o0r6lrammf31oqpao8majloqf6866u8o2nr0msnt6m2tbf9ipckdtvebbo44gqqpvs27miebfn7uo4aa6he                                                                                                                        fqtvfqgfelab9kfi500cbeag6vh33t9c1uabg3hdsl1ppmj2agjje56ircq5kavvr65v6ak5k2t44s8tpdehg2u405t53oab6pvgnrhl2m" length 13                                                                                                                        57
2016/10/12 17:24:41 encryptSegment "mjfcgv9j32dj4i7hao0kpa7u0mvlmhc0qquueqoqcb6crubgpb1m3oeptjktg5qu3nn4dgk10pq7chnmg                                                                                                                        o4h0430q8fqnapgt95o71bms4avi888rhibg7uup65eef6hjm1tqh6c7a63f5j0l12oetgupoohe9n6h4f5s33n9vh3051lo2v7jjp4jf69e74l39qek5                                                                                                                        j5i1og4d57mvad5shlh9mgsihpobutdmtbsvp1d7sfuehumr0rp5opif7p0kk8qp5s4svi58sn0baku25u26kma9a3jsrg7n6rlkmorp49lqlniec7gak                                                                                                                        o8sdjguqu3s9otdd2k0gc8r50tubu3ars3k8sn515c32hdepsnq1mtcolp3tu8lt09onulmakdu1kq7p5iiccvud53lmp797l3bk36copjff3d4r5tbq4                                                                                                                        b7e49uuqma3augqofns48e7jcjqoq1cbpqjqbmk2apdmmac2n9uigbac75pjqd37hfu0q9vbl9o7bo1dqncg2k998f4p530u0h5nu1nhcvbs0q5qvo4q3                                                                                                                        eppc20hh07lbcfr5ot88plf5l5fp17bnprr35680kssj64jchl85cte216gj2gcv259s3b457n1skjvoltkj98eu9j138fk3061ljo7l1u41tsjbhmotc                                                                                                                        t1d9kfhlef0151tsd6o8eiptmbgb9hgmagv1v6b12s48o810v73ub0rkjeuchsams46e962fnc3secnukstialqvanp3iutqfse0v12gpador71d50fhc                                                                                                                        ngcsnkuogev3aagfj0a44qt2br57t20rgt5k9lppp04vs0cmje6i95otn2lat6efal1ab2bheo3qjdq65dp7fkh7neiboag9qkoao6k8cj6sphumvisdh                                                                                                                        j0nvi9m8neo89cdmslug6g1njjj6td42sm0bjkjot1rimrrslcsbpgaf4hcf9ba8su2oe62fa8dfn65jmf455vdfjcfuh52imesmdd9e24njb755mfte9                                                                                                                        ifl855dorflr2saf37v19c6h3i21kk5glqs91eor6u1jkc6q9og4qthhen4t1mfahks70gmlt0l5hp9o4cm9sq4qrof06gmpp6vhp8jdq54nn6t5rq60n                                                                                                                        vjpoljm5a9gutodrcu7n4881ohc7c4r3hfoi7k49qof7vr1cpgvr9l1f0b46u9867nlj3tu4dogmsnq26bbt4ae9p7rjf44i1hk13k3hdiac6fval5eqc                                                                                                                        k75ekeioqlliufpaop20a4j8ea7nlnofr58gsa438dd10dporjlgsl219e11cnult8spfr76lnd9319vg1id1r4c5e9eqmpeni4p27hqok" length 13                                                                                                                        57
2016/10/12 17:24:41 encryptSegment "bhvpeq21fv2iu75v196vi3b93hh444gajolkvpdk0i1r8siq5p6bjbf4h8l1kgfnmc9o8uo0i24usukqt                                                                                                                        1q773l5i234bsnufjc2a10idkq21e9pkndgs49g1o7pukemt81rcr4nvqlt64fu39kkf3fhnhcer699cvs50cvv13rhr391rg09g8jsnac6can7ap6lbm                                                                                                                        vkntcligos66kveq89m7b3q3hqr0s0ke7flbcns11eldt6pkldj0u9n0dkv475g2ob4k6kolgbk6qjhpie4e403sa3596grr1r4ie5ur3mmmjne1i354f                                                                                                                        mldkbdvsd6h1qbmt3i6v2nkrt80d7591r4nda4mghlsk49li9a1aohksp1v9b0smvbucsi6grb34krjae46moaifefi2cc2bvjahmmtocmuh4k68m6chg                                                                                                                        c4ajkmpl3mhijuekl0gamv3p2n2o44ir9hfsobsklpjfsblegtuo4lhq5ervkkr5j6bjvod95an0bvtgg0me9330vo5q1i6beqjr7o2ffv75tgd34q6q6                                                                                                                        5ijhss8d0kun6brjmjqvlc387qdqho3jv2o0irqh226naf5jm6t57364ou2o1cv2jjlrjdjm1fo5aib13t6s82ccklvh1mmglvmq5fa9n5qivg310emt6                                                                                                                        p7ib07t5tj9ciqhff1hi20en2s1csak8cmb9vfr1ads0f8uadq6qso2pqa2m4sebjou64ocg5l27iarougj7tbfiod6ft17hb73ofjnvto158maqberut                                                                                                                        bplvk33md1dnec88c9q90iv8q93ee4mrn8kv8gk7bik1hdi2i3duomkspr261tn527en0f9r45an476u4l0i89enase3q5g8grd3r4o90jdlc05ghfl63                                                                                                                        pe89bs6a2bdked1jhu44987eg30lkt6177lvnn4sqummirn88o1j3pdbt5iabdkm8ga84lj2bvu83b6v4ko8cm40cknj6neabu3t8msl6mncoqroidnkh                                                                                                                        v0mo3fjepkg70lr72i650g1oes3ab56kmaodavrjba3d6f1n4frhvfjlniaq21ombfhr5jgk4qofmatub1n4rtbteqc9sm7bo00t932j6p991kcinbpcq                                                                                                                        voq1pr8r9ot2jt0aqjdsbhs6eb8q8su25mcjs0f4k8b7q3t4vrcv7tsnbugeg8r8gbvb19t1elqtvnqcgor3a5fb3dphiqmratpkk4rpmehng0sq35q7b                                                                                                                        ivk74hdrshfkhfsoe849rr68vpu433bnt6gho4jktfh53uq5h19lbjosa7mhpug1lo9scksmr029lsiatco73c8iitpgt1npc35litovcdgop6qt3emag                                                                                                                        oc9057fhvh90vpsjlf2mu0j6afmqbfmjn5u4c9hjkn685770r0ikls4r3md55k67epfkp9mccluoavkn8cbk3tv3ijkac3mealmsqn81g1scdk5us6d8i                                                                                                                        hoio2o0op6094llfjigj5rt5moqqp4e9c17vl7lhrclguo7t21rav2vgg5765sjl9tgmsgjrft8upnjkp6roi39r6jvn2b39cgckohib6dqppa9ptguba                                                                                                                        o41q45ifj4m859m56tro6oomspnkbaarf9ge5nogn8g899n2iv4a5jfpqukm1omjtmo1i4dbii5tqkk49e2q20e4bm4e9f1g4eb2o2sou6j4unq0025va                                                                                                                        tcjv0stot23bmt3os2cotj6mkaqkngdnhkmdredoggc4u5mr0dn72bs9v3qm8mfvhl47af82nmegkd8upjk4me4b8i7p4pjqjo8ve67l2ui0l27nl23i8                                                                                                                        jjjjp574vqhiup8oggigb0gn49mmv713n15f0cmgdnig0k3m0hsapbkobevn6abe62pg0qm95t9sf0d5od62i33rmd3nuomkeucppoqk2fvo8npe92bm8                                                                                                                        a4fgommejskh73oh3jvu9kuk2h055cq5jsv11nd5q73q5b8kd0b1qubc7heb6n0t4vhcnubh46a3q05luqd9k8igcslm9ejphsllm8d4sf1fm366pdf62                                                                                                                        rq61nam306coscoth50bot2g1e6r8go8vfs3ppuu38ojl8d299598cfj52rj7mg93avea7idnr6ea7fs14av93i81ceabndtli36hor5vm" length 21                                                                                                                        76
2016/10/12 17:24:41 EME operates on 1 to 128 block-cipher blocks, you passed 137
panic: EME operates on 1 to 128 block-cipher blocks, you passed 137

**Comment by ncw** [comment]
What have you got in your crypt config? It looks like it is somehow pointed at itself? 

When using crypt you have one remote say acd: which you point at unencrypted acd, then you make a second crypt remote called encryptedacd: with the remote parameter 'acd:secret'. 

Have you got it set up like that?

**Comment by liberianbarcode** [comment]
I feel terrible now.

You are completely right. I had created an unencrypted ACD and an encrypted remote, but I did point the encrypted remote to itself instead of the unencrypted acd.

I know the main fault here is that my English is not very good, but maybe you consider building in a check that prevents this kind of mistake from happening in the future. 

Thank you very much for your time and for helping me fix the problem.
